stable/kube-state-metrics/templates/stsdiscovery-role.yaml

```diff
@@ -19,11 +19,9 @@ rules:
 - apiGroups:
   - apps
   resourceNames:
-  - {{ template "kube-state-metrics.fullname" . }}
+  - kube-state-metrics
```

Comment:
why is this change needed?

Reply:
It really was't - bad merge on my part. I've fixed it up - also found a few others. There should have been no changes to stsdiscovery-role.yaml :(